Box.V2.Test.Integration/BoxMetadataManagerTestIntegration.cs

```diff
@@ -55,6 +55,20 @@ public class BoxMetadataManagerTestIntegration : BoxResourceManagerTestIntegrati
             Assert.IsTrue(result, "Failed to delete file metadata");
         }
 
+        [TestMethod]
+        public async Task Metadata_DeleteTemplate_LiveSession()
+        {
+            string templateKey = "testtemplate";
+            string displayName = "Test Template";
+            string scope = "enterprise";
+
+            var templateToCreate = new BoxMetadataTemplate() { TemplateKey = templateKey, DisplayName = displayName, Fields = null, Hidden = true, Scope = scope };
+            await _client.MetadataManager.CreateMetadataTemplate(templateToCreate);
+
+            var templateIsDeleted = await _client.MetadataManager.DeleteMetadataTemplate(scope, templateKey);
+            Assert.IsTrue(templateIsDeleted, "Failed to delete metadata template");
```

Comment:
Can we also add a call to get the metadata template after it's been deleted to ensure it's gone?

Reply:
@mattwiller, just updated, thanks!